modules/ROOT/pages/plugins/premium/tinymcespellchecker.adoc

```diff
@@ -77,10 +77,6 @@ include::partial$plugins/tinymcespellchecker-content-langs.adoc[]
 
```

Comment:
I can't comment on it sorry, but the `misc/spellchecker-languages.adoc` line above has the wrong heading level which is causing an issue with the ToC. It needs to be this instead:
```
include::partial$misc/spellchecker-languages.adoc[leveloffset=+2]
```

Reply:
Done

Reply:
@lnewson Hi, can you please resolve this conversation if the changes made are correct ? Thank you

Reply:
Could have sworn I resolved this sorry @hamza0867.

Reply:
No worries 😄